controllers/constant/namespace.go

```diff
@@ -16,35 +16,67 @@
 
 package constant
 
+const NSRestrictedSubscription = `
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ibm-namespace-scope-operator-restricted
+  namespace: {{ .MasterNs }}
+  annotations:
+    version: {{ .Version }}
+spec:
+  channel: {{ .Channel }}
+  installPlanApproval: Automatic
+  name: ibm-namespace-scope-operator-restricted
+  source: opencloud-operators
+  sourceNamespace: openshift-marketplace
```

Comment:
Could we change Nss's `source` and `sourceNamespace` to `{{ .CatalogSourceName }}` and `{{ .CatalogSourceNs }}` that is matching with CsSubscription?